test/TestsHelper.js

```diff
@@ -410,8 +447,21 @@ TestsHelper.setup = {
       },
       (next) => {
         this.clearProjectSetup(null, next);
+      },
+      (next) => {
+        this.clearSupposeDebugFile(null, next);
       }
     ], done);
+  },
+
+  startMockServer(options, cb, done) {
+    mockServer(options, (err, server) => {
+      if (err) {
+        return done(err);
+      }
+
+      cb(server);
+    });
```

Comment:
Having two callbacks is confusing the least. If an error doesn't occur, then `done` won't be called at all... Also, you're breaking standard node.js convention to pass error as first argument to callback and data as second.

Reply:
Reverting to use directly the mockServer function